EnglishTranslation-master/SKILL.tsv

```diff
@@ -4719,7 +4719,7 @@ SKILL_20150717_004718	Attack: -#{CaptionRatio}#% {nl}Increased defense equal to
 SKILL_20150717_004719	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio}#%{nl}Duration: #{CaptionRatio2}# seconds{nl}AoE Attack Ratio: #{SkillSR}#	공격력 #{SkillAtkAdd}#{nl}특성 공격력 #{CaptionRatio}#%{nl}지속시간 #{CaptionRatio2}#초{nl}광역 공격 비율 #{SkillSR}#				
 SKILL_20150717_004720	Summon the devil of card inserted in Grimore. The attack and defense of summoned devil will be decided according to character's INT, SPR. Grimore can only equip Devil type monster card.	그리모어에 삽입한 카드의 악마를 소환합니다. 소환된 악마의 공격력, 방어력은 캐릭터의 지능, 정신 능력치의 영향을 받습니다. 그리모어에는 악마형 몬스터 카드만 장착할 수 있습니다.	SKILL_20150414_003717	Summon the devil of card inserted in Grimore. Grimore can only equip Devil type monster card.	그리모어에 삽입한 카드의 악마를 소환합니다. 그리모어에는 악마형 몬스터 카드만 장착할 수 있습니다.	
 SKILL_20150717_004721	Summons a Devil-type boss monster{nl}Devil's Additional Attack: #{CaptionRatio}#{nl}Devil's Additional Defense: #{CaptionRatio2}#	악마형 보스 소환{nl}악마 추가 공격력 #{CaptionRatio}#{nl}악마 추가 방어력 #{CaptionRatio2}#	SKILL_20150714_004172	Summons a Devil-type boss monster{nl}Devil's Abilities: +#{CaptionRatio}#%	악마형 보스 소환{nl}악마 능력치 #{CaptionRatio}#% 증가	
-SKILL_20150717_004722	Movement Speed -#{CaptionRatio}#{nl}Magic Circle Duration: #{CaptionRatio2}# seconds{nl}{#339999}{ol}[Slow]{/}{/} Duration: 5 seconds	이동 속도 #{CaptionRatio}# 감소{nl}마법진 지속시간 #{CaptionRatio2}#초{nl}{#339999}{ol}[슬로우]{/}{/} 지속시간 5초	SKILL_20150401_003223	Movement speed - #{CaptionRatio}#%{nl}Magic circle duration #{CaptionRatio2}# seconds{nl}{#339999}{ol}[Slow]{/}{/} duration 5 seconds	이동 속도 #{CaptionRatio}#% 감소{nl}마법진 지속시간 #{CaptionRatio2}#초{nl}{#339999}{ol}[슬로우]{/}{/} 지속시간 5초	
+SKILL_20150717_004722	Movement Speed -#{CaptionRatio}#{nl}Magic Circle Duration: #{CaptionRatio2}# seconds{nl}{#339999}{ol}[Slow]{/}{/} Duration: 5 seconds	이동 속도 #{CaptionRatio}# 감소{nl}마법진 지속시간 #{CaptionRatio2}#초{nl}{#339999}{ol}[슬로우]{/}{/} 지속시간 5초	SKILL_20150401_003223	Movement speed - #{CaptionRatio}#%{nl}Magic Circle Duration #{CaptionRatio2}# seconds{nl}{#339999}{ol}[Slow]{/}{/} duration 5 seconds	이동 속도 #{CaptionRatio}#% 감소{nl}마법진 지속시간 #{CaptionRatio2}#초{nl}{#339999}{ol}[슬로우]{/}{/} 지속시간 5초	
```

Comment:
Is this description in-game? If so, we should standardize like the other descriptions for consistency's sake.

`Movement Speed: -#{CaptionRatio}#%{nl}Magic Circle Duration: #{CaptionRatio2}# seconds{nl}{#339999}{ol}[Slow]{/}{/} Duration: 5 seconds`